### PR TITLE
Update v1.2 release notes

### PR DESCRIPTION
**Pull request by bningOSI**
Hi Team,

This documentation change contains the release note update for OPC UA v1.2.

Let me know if you have any concerns or questions.

Thanks

### CODE REVIEW

File: content/release-notes/release-notes.md [hunk 1]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
 
 ## Overview
 
-This represents the initial standalone release for PI Adapter for OPC UA. This product collects time series data and relevant metadata from an OPC UA (OPC Unified Architecture) server and sends it to configured OMF endpoints such as PI Web API and OSIsoft Cloud Services. PI Adapter for OPC UA can also collect health and diagnostics information. It supports buffering, unsolicited data collection, automatic discovery of available data items on a data source, and various Windows and Linux-based operating systems as well as containerization.
+This represents the release version 1.2.0 for the PI Adapter for OPC UA. This product collects time series data and relevant metadata from an OPC UA (OPC Unified Architecture) server and sends it to configured OMF endpoints such as PI Web API and OSIsoft Cloud Services. PI Adapter for OPC UA can also collect health and diagnostics information. It supports buffering, unsolicited data collection, automatic discovery of available data items on a data source, on-demand/automatic history recovery of data items supporting HistoryReady user access level, and various Windows and Linux-based operating systems as well as containerization.
```
Comment (zryska) [inline content/release-notes/release-notes.md:11]: I believe that we could consider removing the first sentence and change the second to: PI Adapter for OPC UA collects time series...

and start with it.

Comment (zryska) [inline content/release-notes/release-notes.md:11]: We might change the automatic discovery to on-demand discovery of available data items

Comment (zryska) [inline content/release-notes/release-notes.md:11]: on-demand/automatic history recovery, and various Windows...

I wouldn't include this "of data items supporting HistoryReady user access level" that's for in dept description of feature

File: content/release-notes/release-notes.md [hunk 2]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
+### Fixes
+The following issues reported from version 1.1 are fixed in this release.
+
+- Fixed the issue that timestamp and value migth not be correctly updated for the fast sampling rates.
+- Fixed the issue that browse operation can fail when empty result is returned from a server.
+
+### Enhancements
+The following enhancements are added in this release.
+
+- Enhanced user experience to perform data source discovery.
```
Comment (zryska) [inline content/release-notes/release-notes.md:26]: On demand data source discovery feature has been added. or something along those lines

File: content/release-notes/release-notes.md [hunk 3]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
+- Added support for On-demand and automatic history recovery for data items that support HistoryReady user access level.
```
Comment (zryska) [inline content/release-notes/release-notes.md:27]: Here I would also stop with Added support for on on-demand and automatic history recovery.

File: content/release-notes/release-notes.md [hunk 4]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
+- Added support for container creation with Unit of Measure, Min, Max, and Data Source properties.
+- Enabled data collection for enumeration data types from data items that have underlying enumeration types.
+- Added support for collecting data quality information along with the container data.
```
Comment (zryska) [inline content/release-notes/release-notes.md:30]: Adapter now collects data quality information.

File: content/release-notes/release-notes.md [hunk 5]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
+- Fixed the issue that timestamp and value migth not be correctly updated for the fast sampling rates.
+- Fixed the issue that browse operation can fail when empty result is returned from a server.
+
+### Enhancements
+The following enhancements are added in this release.
+
+- Enhanced user experience to perform data source discovery.
+- Added support for On-demand and automatic history recovery for data items that support HistoryReady user access level.
+- Added support for container creation with Unit of Measure, Min, Max, and Data Source properties.
+- Enabled data collection for enumeration data types from data items that have underlying enumeration types.
```
Comment (zryska) [inline content/release-notes/release-notes.md:29]: Added support for enumeration data types.

File: content/release-notes/release-notes.md [hunk 6]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
+
+- Fixed the issue that timestamp and value migth not be correctly updated for the fast sampling rates.
+- Fixed the issue that browse operation can fail when empty result is returned from a server.
+
+### Enhancements
+The following enhancements are added in this release.
+
+- Enhanced user experience to perform data source discovery.
+- Added support for On-demand and automatic history recovery for data items that support HistoryReady user access level.
+- Added support for container creation with Unit of Measure, Min, Max, and Data Source properties.
```
Comment (zryska) [inline content/release-notes/release-notes.md:28]: Added support for collection of Unit of Measure, Minimum and Maximum.

Data source property was supported in the past.

**Review by zryska — APPROVED** [review]

File: content/release-notes/release-notes.md (hunk 1 above)
Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

File: content/release-notes/release-notes.md (hunk 2 above)
Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

File: content/release-notes/release-notes.md (hunk 3 above)
Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

File: content/release-notes/release-notes.md (hunk 4 above)
Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

File: content/release-notes/release-notes.md (hunk 5 above)
Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

File: content/release-notes/release-notes.md (hunk 6 above)
Comment (bning0822) [reply content/release-notes/release-notes.md]: Fixed. Thanks

File: content/release-notes/release-notes.md [hunk 7]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
 
 # Release notes
 
-PI Adapter for OPC UA 1.1
+PI Adapter for OPC UA 1.2
 
 ## Overview
 
-This represents the initial standalone release for PI Adapter for OPC UA. This product collects time series data and relevant metadata from an OPC UA (OPC Unified Architecture) server and sends it to configured OMF endpoints such as PI Web API and OSIsoft Cloud Services. PI Adapter for OPC UA can also collect health and diagnostics information. It supports buffering, unsolicited data collection, automatic discovery of available data items on a data source, and various Windows and Linux-based operating systems as well as containerization.
+PI Adapter for OPC UA collects time series data and relevant metadata from an OPC UA (OPC Unified Architecture) server and sends it to configured OMF endpoints such as PI Web API and OSIsoft Cloud Services. PI Adapter for OPC UA can also collect health and diagnostics information. It supports buffering, unsolicited data collection, on-demand discovery of available data items on a data source, on-demand/automatic history recovery of data items, and various Windows and Linux-based operating systems as well as containerization.
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:11]:
```suggestion
PI Adapter for OPC UA collects time series data and relevant metadata from an OPC UA (OPC Unified Architecture) server and sends it to configured OMF endpoints such as PI Web API and OSIsoft Cloud Services. PI Adapter for OPC UA can also collect health and diagnostics information. It supports buffering, unsolicited data collection, on-demand discovery of available data items on a data source, on-demand or automatic history recovery of data items, and various Windows and Linux-based operating systems as well as containerization.
```

File: content/release-notes/release-notes.md [hunk 8]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
+- Fixed the issue that browse operation can fail when empty result is returned from a server.
+
+### Enhancements
+The following enhancements are added in this release.
+
+- Added on-demand data source discovery.
+- Added support for On-demand and automatic history recovery.
+- Added support for collection of Unit of Measure, Minimum and Maximum.
+- Added support for enumeration data types.
+- Added support for collecting data quality information along with data.
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:30]:
```suggestion
- On-demand data source discovery.
- Support for On-demand and automatic history recovery.
- Support for collection of Unit of Measure, Minimum, and Maximum.
- Support for enumeration data types.
- Support for collecting data quality information along with data.
```

File: content/release-notes/release-notes.md [hunk 9]
```diff
@@ -4,14 +4,31 @@ uid: ReleaseNotes
...
 
 For more information see [PI Adapter for OPC UA overview](xref:PIAdapterForOPCUAOverview).
 
+## Fixes and enhancements
+
+### Fixes
+The following issues reported from version 1.1 are fixed in this release.
+
+- Fixed the issue that timestamp and value migth not be correctly updated for the fast sampling rates.
+- Fixed the issue that browse operation can fail when empty result is returned from a server.
```
Comment (TinaUll) [inline content/release-notes/release-notes.md:21]:
```suggestion
- Timestamp and value might not be correctly updated for the fast sampling rates.
- Browse operation can fail when empty result is returned from a server.
```

**Pull request merged**